lua/github_access.lua

```diff
@@ -61,32 +60,33 @@ function update_cache(auth_md5, status)
     file:write(status)
     file:close()
 
-    res = os.rename(path_orig, path)
+    local res = os.rename(path_orig, path)
     if res == nil then
         os.remove(path_orig)
         return nil
     end
 end
 
 function authenticate(auth)
-    divider = auth:find(':')
+    local divider = auth:find(':')
     local username = auth:sub(0, divider-1)
     local auth_md5 = ngx.md5(auth)
 
-    cached_status = read_cache(auth_md5)
+    local cached_status = read_cache(auth_md5)
 
     if cached_status == nil then
         local res =  ngx.location.capture("/force_github_request/orgs/" .. github_api_company .. "/members/" .. username)
-        ngx.log(ngx.STDERR, "status member \t\t" .. res.status)
 
         update_cache(auth_md5, res.status)
 
         if res.status ~= 204 then
+            ngx.log(ngx.STDERR, 'User ' .. username .. ' not allowed (github auth cache MISS)')
             return  false
         end
         return true
     else
         if cached_status ~= '204' then
+            ngx.log(ngx.STDERR, 'User ' .. username .. ' not allowed (github auth cache HIT)')
```

Comment:
I though this one and the log above would be super useful if we want to know who refuses the access (cache or github)